events/wc_realm_events.txt

```diff
@@ -545,4 +545,158 @@ character_event = {
 		# Clear flags and other stuff
 		clear_settler_flag_effect = yes
 	}
+}
+
+### Ymiron Awakened ###
+character_event = {
+	id = WCRLM.799
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	immediate = {
+		# Saves ROOT's liege for future use
+		if = {
+			limit = { independent = no }
+			liege = { save_event_target_as = target_liege }
+		}
+		
+		# Creates Ymiron
+		create_character = {
+			name=Ymiron
+			dynasty=70000
+			dna="anolmcpmehb"
+			culture=vrykul religion=ROOT
+			attributes = { martial=6 diplomacy=6 stewardship=4 intrigue=7 learning=7 }
+			trait=skilled_tactician
+			trait=class_warrior_7
+			trait=wroth trait=proud trait=ambitious trait=arbitrary trait=paranoid
+			trait=creature_vrykul
+			random_traits = yes
+			age = 300
+		}
+		new_character = {
+			save_event_target_as = target_ymiron
+		}
+		event_target:target_ymiron = {
+			# Gives bloodline
+			create_bloodline = {
+				type = ymiron
+			}
+			
+			# Angerboda becomes your spouse
```

Comment:
`event 500` fires for Angerboda

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768479-45d46980-2f27-11ea-81f3-af521bd6506b.png)
![image](https://user-images.githubusercontent.com/46576860/71768480-4a008700-2f27-11ea-9120-ae3fc00cdb10.png)


</details>

Reply:
And what?

Reply:
And maybe you should think at least once in a while.
This event shouldn't fire for landless character, shouldn't give gold/prestige to Angerboda instead of Ymiron, probably shouldn't fire at all for either of them, since it's not new marriage.

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/33c1a3e573edf90c9e5e78ac3def329593a61f43.